medusa/post_processor.py

```diff
@@ -367,14 +367,16 @@ def rename_associated_file(new_path, new_basename, filepath):
             new_extension = 'nfo-orig'
 
         elif is_subtitle(filepath):
-            sub_code = filepath.rsplit('.', 2)[1]
-            code = sub_code.lower().replace('_', '-')
-            if from_code(code, unknown='') or from_ietf_code(code, unknown=''):
-                # TODO remove this hardcoded language
-                if code == 'pt-br':
-                    code = 'pt-BR'
-                new_extension = code + '.' + extension
-                extension = sub_code + '.' + extension
+            sub_code = filepath.rsplit('.', 2)
+            # len != 3 means we have a subtitle without language
+            if len(sub_code) == 3:
+                code = sub_code[1].lower().replace('_', '-')
+                if from_code(code, unknown='') or from_ietf_code(code, unknown=''):
+                    # TODO remove this hardcoded language
+                    if code == 'pt-br':
```

Comment:
pt-br > 3 ?

Reply:
Nope, that's something different. Try to `filepath.rsplit('.', 2)` and you will understand what's going on.

Reply:
we also support `en` , `pt`, and `it` and language codes

Reply:
That should be fine, see the tests.